easybuild/easyblocks/w/wien2k.py

```diff
@@ -186,23 +204,57 @@ def configure_step(self):
             else:
                 raise EasyBuildError("Required FFTW dependency is missing")
             qanda.update({
-                 '(not updated) Selection:': comp_answer,
+                 ') Selection:': comp_answer,
                  'Shared Memory Architecture? (y/N):': 'N',
                  'Set MPI_REMOTE to  0 / 1:': '0',
                  'You need to KNOW details about your installed  MPI and FFTW ) (y/n)': 'y',
-                 'Please specify whether you want to use FFTW3 (default) or FFTW2  (FFTW3 / FFTW2):' : fftw_spec,
-                 'Please specify the ROOT-path of your FFTW installation (like /opt/fftw3):' : fftw_root,
-                 'is this correct? enter Y (default) or n:' : 'Y',
+                 'Please specify whether you want to use FFTW3 (default) or FFTW2  (FFTW3 / FFTW2):': fftw_spec,
+                 'Please specify the ROOT-path of your FFTW installation (like /opt/fftw3):': fftw_root,
+                 'is this correct? enter Y (default) or n:': 'Y',
+                 'LIBXC (that you have installed before)? (y,N):': '',
```

Comment:
We actually have `libxc`, so you could add it as a dependency?

Reply:
done